### PR TITLE
index.html and css added

### PR DESCRIPTION
**Pull request by mihrab34**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+		<a href="">
+			<h3>What will your Verse be?</h3>
+			<p>Everyone has something to share. See how people are using iPad in their unique ways to add to the world story.</p>
+			<p id="verse">Watch the "your Verse" film and explore the stories </p>
+		</a>
+    </section>
+</main>
+
+<section class="row">
+	<div class="grid">
```
Comment (Redvanisation) [inline index.html:34]: The `<section>` elements should have a header element as a direct child `<h2>` to `<h6>` before a div

File: index.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+			<input type="text" name="search" id="input" >
+		</div>
+	</div>
+
+    <section class="hero">
+		<a href="">
+			<h3>What will your Verse be?</h3>
+			<p>Everyone has something to share. See how people are using iPad in their unique ways to add to the world story.</p>
+			<p id="verse">Watch the "your Verse" film and explore the stories </p>
+		</a>
```
Comment (Redvanisation) [inline index.html]: The main title should be an `<h1>` tag for a better SEO and it is better to place an `<a>` inside each of the three hero elements (h3 and the two P tags)

**Review by Redvanisation — CHANGES_REQUESTED** [review]
### Hello @mihrab34   🌟 

### Great job on that project I really like your work, however before we move on we should do some small improvements:

1. Please **fix all the HTML validation errors**: https://validator.w3.org/#validate_by_input

2. The main heading (the `<h3>`) should be displayed on one line.

3. The main title should be an `<h1>` tag for a better SEO and it is better to place an <a> inside each of the three hero elements (h3 and the two P tags).

###  _After making the required changes please submit a new code review request_

### **Goodluck!** 🍀

**Review by stratospherique — CHANGES_REQUESTED** [review]
### Great work :clap: . Unfortunately, you still need to implement the following changes in order for us to approve your project. check below :arrow_down: 

- Fix validator errors.

### That's it. When finished, kindly resubmit another code review request :pray: .

**Comment by mihrab34** [comment]
All validator errors have been fixed. Adding an empty heading as a direct child of the section element is not necessary for this webpage.

**Review by stratospherique — APPROVED** [review]
### Great Work :clap: . You made all the necessary changes :100: . Your project is approved and you can merge to master :+1: :+1: :+1:

**Pull request merged**